charts/studio/templates/persistentvolume.yaml

```
@@ -8,8 +8,13 @@ spec:
   {{- with .Values.global.blobvault.persistentVolume.accessModes }}
   {{- toYaml . | nindent 4 }}
   {{- end }}
-  {{- if .Values.global.blobvault.persistentVolume.storageClassName }}
-  storageClassName: {{ .Values.global.blobvault.persistentVolume.storageClassName }}
+
+  {{- $pvc := (lookup "v1" "PersistentVolumeClaim" .Release.Namespace "blobvault") | default dict }}
+  {{- $pvcSpec := (get $pvc "spec") | default dict }}
+  {{- $storageClassName := (get $pvcSpec "storageClassName") | default $.Values.global.blobvault.persistentVolume.storageClassName }}
+{{/*  {{- $storageClassName :=  $.Values.global.blobvault.persistentVolume.storageClassName }}*/}}
```

```suggestion
{{/*  {{- $storageClassName :=  $.Values.global.blobvault.persistentVolume.storageClassName }}*/}}
```
Let's remove the comment?